### DISCUSSION TITLE
Changing the changelog process of ruby-jwt?

### DISCUSSION THREAD

**Discussion by anakinj** [jwt/ruby-jwt#461] [Ideas]
@excpt What do you think if we would move the responsibility of adding a descriptive Changelog entry to the author of the PR.

I really like the approach of ruby-grape https://github.com/ruby-grape/grape/blob/master/CONTRIBUTING.md

So instead of generating the changelog based on PRs merged the changelog would build itself between releases.

**Comment by excpt** [comment]
This is great. I think we should adopt this. 👍

**Reply by anakinj** [reply]
You think it's ok to drop the "Closed issues" header and only have "Features" and "Fixes".

I could create something based on the already merged PRs and write some kind of documentation on how to make these changelog entries.

After this we can start bugging PR makers to update the changelog and mabye borrow some functionality around the danger bot that automatically reminds the author of the PR about the changelog entry.
